sqlmesh/core/engine_adapter/base.py

```diff
@@ -87,13 +87,18 @@ def requires_condition(self) -> bool:
 
 class CatalogSupport(Enum):
     UNSUPPORTED = 1
-    REQUIRES_SET_CATALOG = 2
-    FULL_SUPPORT = 3
+    SINGLE_CATALOG_ONLY = 2
+    REQUIRES_SET_CATALOG = 3
+    FULL_SUPPORT = 4
```

Comment:
These enums are not persisted anywhere so they can be switched around like this.